src/protobot_mapping/config/robots/protobot.yaml

```diff
@@ -1,18 +1,18 @@
-map_frame_id: odom
+map_frame_id: map
```

Comment:
why map? map is discontinuous, you just can't tell because there's no sensor noise